docs/src/main/paradox/migration-guide/index.md

```diff
@@ -4,11 +4,7 @@
 
 @@@ index
 
-* [migration-guide-10.2.x](migration-guide-10.2.x.md)
-* [migration-guide-10.1.x](migration-guide-10.1.x.md)
-* [migration-guide-10.0.x](migration-guide-10.0.x.md)
-* [migration-guide-2.4.x-10.0.x](migration-guide-2.4.x-10.0.x.md)
-* [migration-from-spray](migration-from-spray.md)
-* [migration-from-old-http-javadsl](migration-from-old-http-javadsl.md)
+* If you use an old version of Akka HTTP or Spray, you should first follow the Akka HTTP migration guide (TODO add links to Akka docs)
```

Comment:
The validatePullRequest sbt task fails if I include URLs. I haven't yet worked out how to relax the check.

Failures like:
```
[info] `docs / Compile / ./ paradox` failed because of an underlying problem during task execution:
[info] ```
[info]   _: com.lightbend.paradox.markdown.Index$LinkException: Unknown page [https://doc.akka.io/docs/akka-http/current/release-notes/index.html] linked from [release-notes/index.md]
[info] ```
[info] </details>
```

Reply:
I'm guessing but I think we need https://github.com/apache/incubator-pekko-http/issues/27 - after this is done, I can try adding these links